Adding message prop usage and example on documentation (#37)
vtex-apps/product-price: docs/README.md

},
```

Every block in this app only has three props in common:

| Prop name          | Type      |  Description | Default value |
| --------------------| ----------|--------------|---------------|

> | `markers`           |`[string]` | IDs of your choosing to identify the block's exported messages and customize them using the admin's Site Editor. Learn how to use them accessing the documentation on [Using the Markers prop to customize a block's message](https://vtex.io/docs/recipes/style/using-the-markers-prop-to-customize-a-blocks-message). |`[]`|

Reviewer: ```suggestion
| `markers`           |`array` | IDs of your choosing to identify the block's rendered message and customize it using the admin's Site Editor. Learn how to use them accessing the documentation on [Using the Markers prop to customize a block's message](https://vtex.io/docs/recipes/style/using-the-markers-prop-to-customize-a-blocks-message). Notice the following: a block's message can also be customized in the Store Theme source code using the `message` prop. |`undefined`|
```

|  `blockClass`  |  `string`  |  Block  ID  of your choosing to  be  used  in [CSS  customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property).  |  `undefined`  |

> |  `message`  |  `string`  |  Message variables exported by each block that can be edited in Site Editor as well  |  `undefined`  |

Reviewer: ```suggestion
|  `message`  |  `string`  |  Defines the block's rendered text i.e. the block message. Instead of using this prop, you can also define which text message a block will render on the UI using the admin's Site Editor and the `markers` prop.  |  `undefined`  |
```

    "markers": [
      "highlight"
    ],
    "blockClass": "summary",

>     "message": "{sellingPriceValue}"

Reviewer: ```suggestion
    "message": "Selling price"
```

Author: > Thank you so much, Hugo! I was about to update this when the notification came up ❤️

You are welcome, sorry about my poor writing.

About your suggestion ("message": "Selling price"), I think that will be a good choice to put the variable itself at the example, to really show the correct usage, I struggle a bit trying to make this work kkkkkk.

Reviewer: No needs to apologize!!! It was a great contribution 💯 

Since this is a practical example I don't see why we need to use the variable itself. In practice, users will use the `message` prop to declare which message text they want to be rendered, right?

Author: Correct, but I think it will be great to show the possibility to use the variables because the product-price blocks are heavily dependent by them.

Reviewer: Why do you think we should use the variable as the `message` prop value? I mean, the product price blocks are heavily dependent on the variables because of the `markers` prop. I don't see the link between `message`  prop and the variables.

Author: I don't know the motive behind this feature.  I used that way, I edit the message showed with the variables, because not always the clients wanted the default message displayed. E.g, If I want to add "OFF" string after {sellingPriceValue} I just use this: "message": " {sellingPriceValue} OFF". In that way, I'm free to customize the messages of product-price as I want.

Reviewer: I believe @iaronaraujo can confirm this!

Reviewer 2: I think it is better to merge them and use the example 
"message": "Selling price: {sellingPriceValue}"
as @hugocostadev mentioned!
@guerreirobeatriz in this case it is important to show the variable so the people using this will know that you can use variables on this prop the same way you can on site editor!

Author: I will update the changes. Thanks.

  }
},
```

> ### Step 3 - Editing the block's message on Site Editor or in the message prop

Reviewer: ```suggestion
### Step 3 - Editing the block's messages
```

> 
> Every Product Price's block uses the [ICU Message Format](https://format-message.github.io/icu-message-format-for-translators/), making it possible to fully edit the text messages and the variables displayed by each block.

Reviewer: ```suggestion

Every Product Price's block uses the [ICU Message Format](https://format-message.github.io/icu-message-format-for-translators/), making it possible to fully edit the text message variables displayed by each block.
```

> To edit the messages with the block message prop follow the exemple in the above section using the avalible variables bellow. 
> 
> To edit the messages in the Site Editor access your VTEX account's admin, open the Site Editor and look for the Product Price's blocks. 

Reviewer: ```suggestion
The variable values are rendered according to the context wrapping the block, meaning that this last one uses product data to render the price accordingly. 

It is possible, however, to define which message texts a block should render on the UI using the `markers` or the `message` props, as explained previously. 

The text value passed to the `message` prop will be automatically taken into account when the block is rendered. 

The `markers` prop, in turn, needs an extra configuration in the admin's Site Editor to properly work. When using it, do not forget to access the [Using the Markers prop to customize a block's message](https://vtex.io/docs/recipes/style/using-the-markers-prop-to-customize-a-blocks-message/) documentation. 

![Product-Price-Site-Editor-gif](https://user-images.githubusercontent.com/52087100/78073694-bdbffd80-7377-11ea-9262-40854dccdd53.gif)

In addition to that, keep in mind the message variables for each block since you will need them to edit the desired messages using the admin's Site Editor:
```

> ![Product-Price-Site-Editor-gif](https://user-images.githubusercontent.com/52087100/78073694-bdbffd80-7377-11ea-9262-40854dccdd53.gif)

Reviewer: ```suggestion
```

Reviewer: Don't know why, but I'm not being able to delete the next paragraph as well. Could you do this for me? 😃

Author: Sure. Done!
